.github/workflows/stale.yml

```diff
@@ -0,0 +1,48 @@
+name: 👓 Handle stale issues
+on:
+  schedule:
+  - cron: "30 2 * * *"  # Runs every day at 2:30 AM UTC
+
+permissions:
+  contents: read
+
+jobs:
+  stale:
+    permissions:
+      issues: write  # for actions/stale to close stale issues
+    if: github.repository_owner == 'GIScience'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/stale@v9
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+
+        stale-issue-message: >
+          The Openrouteservice project highly values your report and would love to see it addressed.
```

Comment:
```suggestion
          We highly value your report and would love to see it addressed.
```

Reply:
I feel like this makes it a bit more personal but less professional.